content/en/docs/installation/supported-releases.md

```diff
@@ -17,36 +17,39 @@ release every two months.
 
 ## Supported releases {#supported-releases}
 
-| Release | Release Date |        EOL        | [Supported Kubernetes versions][s] |
-| ------- | :----------: | :---------------: | :--------------------------------: |
-| [1.3][] | Apr 08, 2021 |   [~][]Aug 2021   | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
-| [1.2][] | Feb 10, 2021 | [~][]Jun 11, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| Release | Release Date |     EOL      |    [Supported Kubernetes versions][s]    |
+|---------|:------------:|:------------:|:----------------------------------------:|
+| [1.4][] | Jun 15, 2021 | Oct 13, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21, 1.22 |
+| [1.3][] | Apr 08, 2021 | Aug 11, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21, 1.22 |
```

Comment:
nitpick: sorry, I don't think we can actually commit to 1.22 for _currently_ supported releases. As discussed in the Jetstack weekly meeting earlier, we might not actually backport ingress v1 support to 1.4, which would mean that 1.4 won't support 1.22

so, 1.5 _for sure will_ support 1.22 and that should be in "Upcoming releases". But we can't say yet that 1.4 and 1.3 will.

Reply:
You are right. I removed (228f692) the 1.22 mentions in all versions except for 1.5.